test/saithrift/tests/sail3.py

```diff
@@ -1477,3 +1477,194 @@ def runTest(self):
             self.client.sai_thrift_remove_router_interface(rif_port_id)
             self.client.sai_thrift_remove_virtual_router(self.vr_id)
             print "END OF TEST"
+@group('lag')
+@group('l3')
+class L3MultipleEcmpLagTest(sai_base_test.ThriftInterfaceDataPlane):
+    total_changing_ports = 15
```

Comment:
its the changing ports 
the first port is not changing 
what I mean by changing is 
that it will start as Rif and will change to lag